externals/llvm-external-projects/torch-mlir-dialects/lib/Conversion/TcpToLinalg/PopulatePatterns.h

```diff
@@ -0,0 +1,23 @@
+//===------------------------------------------------------------*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+// Also available under a BSD-style license. See LICENSE.
+//
+//===----------------------------------------------------------------------===//
+
+#include "mlir/Transforms/DialectConversion.h"
+
+namespace mlir {
+namespace tcp_to_linalg {
```

Comment:
If you plan to move this into MLIR core maybe this should follow LLVM coding conventions, i.e. camel case instead of snake case.